content/en/docs/contribute/style/style-guide.md

```diff
@@ -44,26 +44,24 @@ The English-language documentation uses U.S. English spelling and grammar.
+When you refer specifically to interacting with an API object, use [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as Pascal Case. You may see different capitalization, such as "configMap", in the  [API Reference](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/). It's better to use UpperCamelCase and call that "ConfigMap" instead when writing general documentation. 
 
-You may use the word "resource", "API", or "object" to clarify a Kubernetes resource type in a sentence. Note that Kubernetes resource names are capitalized. However, capitalization alone is not a reliable way to identify Kubernetes resource types in a sentence. 
+When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).
 
-The sentence "However, pods may have multiple volume mounts" describes general concepts, and uses lower case. In contrast, the sentence "The Pod object includes a Volumes field" is talking about API objects, and uses UpperCamelCase. You may also use [Code Style](#code-style) to distinguish API objects. For example, "The `Pod` object includes a `volumes` field."
-
-Capitalization varies in the Kubernetes API. However, API objects are documented in UpperCamelCase. For example, you may see "configMap" in the [API reference](https://kubernetes.io/docs/reference/). It's fine to call that "ConfigMap" instead when writing general documentation. 
+You may use the word "resource", "API", or "object" to clarify a Kubernetes resource type in a sentence. Note that Kubernetes resource names are capitalized. However, capitalization alone is not a reliable way to identify Kubernetes resource types in a sentence. Review the related guidance on [Code Style](#code-style-inline-code-commands-api-objects)
 
 Don't split the API object name into separate words. For example, use
 PodTemplateList, not Pod Template List.
 
-{{< table caption = "Do and Don't - API objects" >}}
+The following examples focus on capitalization. Review the related guidance on [Code Style](#code-style-inline-code-commands-api-objects) for more information on formatting API objects. 
+
+{{< table caption = "Do and Don't - Upper camel case for API objects" >}}
 Do | Don't
 :--| :-----
 A PodList is a list of pods. | A Pod List is a list of pods.
-HostPath is a type of Volume. | Hostpath is a type of volume.
-`HostPath` is a type of Volume. | Hostpath is a type of volume.
-Every `ConfigMap` is part of a `Namespace` | Every config map object is part of a Namespace object. 
-`VolumeAttachment` objects are non-namespaced. | VolumeAttachments aren't namespaced.
-For managing confidential data, consider using a `Secret`. | For managing confidential data, consider using a secret.
+There is a HostPath field is the Volume resource. | Hostpath is a type of volume.
```

Comment:
```suggestion
There is a `hostPath` field in the Volume resource. | Hostpath is a type of volume.
```

Reply:
Sorry for the nit, but I am not sure about the "Do not" example because you begin a sentence
with the volume type name. What is the name of the volume type? "hostPath" is the volume plugin type?

Reply:
I corrected is/in, and harmonized the do/don't examples. 

> There is a HostPath field in the Volume resource. | There is a hostpath field in the volume resource.